af/fastpath/debian/detector.service

```diff
@@ -0,0 +1,39 @@
+[Unit]
+Description=OONI Detector Webapp
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+ExecStart=/usr/bin/detector --webapp
+Restart=on-abort
+Type=simple
+RestartSec=2s
+WorkingDirectory=/var/lib/detector
+
+User=fastpath
+Group=fastpath
+ReadOnlyDirectories=/
+ReadWriteDirectories=/proc/self
+ReadWriteDirectories=/var/lib/detector
+
+StandardOutput=syslog+console
+StandardError=syslog+console
+
+PermissionsStartOnly=true
+LimitNOFILE=65536
+
+# Hardening
+CapabilityBoundingSet=CAP_SETUID CAP_SETGID
+SystemCallFilter=~@clock @debug @cpu-emulation @keyring @module @mount @obsolete @raw-io @reboot @swap
+NoNewPrivileges=yes
+PrivateDevices=yes
+PrivateTmp=yes
+ProtectHome=yes
+#ProtectHome=read-only
+ProtectSystem=strict
+ProtectSystem=full
```

Comment:
Is `ProtectSystem=full` overriding or augmenting `ProtectSystem=strict`?

Reply:
It's overriding it, I should remove the other.
https://www.freedesktop.org/software/systemd/man/systemd.exec.html#Sandboxing